Cooper/ClientApp/tsconfig.json

```diff
@@ -17,6 +17,16 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "paths": {
+    "@app*": ["src.v2/app/*"],
+    "@components*": ["src.v2/app/components/*"],
```

Comment:
"@components/*": ["src.v2/app/components/*"]

and for others the same

then you can import only from "@components" without index

If you'll have another problems, please, write me